ui/src/dashboards/actions/thunks.ts

```diff
@@ -223,7 +232,7 @@ export const createDashboardFromTemplate = (
 
     const dashboards = normalize<Dashboard, DashboardEntities, string[]>(
       resp.data.dashboards,
-      schemas.arrayOfDashboards
+      arrayOfDashboards
```

Comment:
I actually like the schemas.whatever notation, since it expresses that `arrayofDashboards` is not an actual array of dashboards but defining the schema. It's not that big of a deal.

Reply:
@ebb-tide I liked it too.  Sadly, `importing * as foo` should be used sparingly.  I suppose we could go through and rename everything `arrayOfDashboardsSchema` or some such thing. But that seems a bit long.